packages/rspack-cli/tests/api/type/js-api-cjs.test.ts

```diff
@@ -0,0 +1,31 @@
+import rspackCjsRequire, {
```

Comment:
What's the difference between this cjs case and the esm case? They seem to be the same.

Reply:
one is mts, the other is ts

Reply:
get👌